src/go/k8s/controllers/redpanda/cluster_controller.go

```diff
@@ -40,6 +41,12 @@ const (
 	dataDirectory	= "/var/lib/redpanda/data"
 	fsGroup		= 101
 
+	configDir		= "/etc/redpanda"
+	configPath		= configDir + "/redpanda.yaml"
+	configuratorDir		= "/mnt/operator"
+	configuratorScript	= "configurator.sh"
```

Comment:
oh this is cool; can we do a prezo sometime next week, perhaps next wednesday on the active design and impl? perhaps in `panda chat`?